src/interfaces/swig/Classifier.i

```diff
@@ -73,10 +73,22 @@
 
 /* Include Class Headers to make them visible from within the target language */
 %include <shogun/machine/Machine.h>
+
+/** Instantiate RandomMixin */
+%template(RandomMixinMachine) shogun::RandomMixin<shogun::CMachine, std::mt19937_64>;
+
```

Comment:
these are extremely ugly, but I have to specialize every RandomMixin right after the definition of the base class, so that derived classes can recognize it

Reply:
mmmh

imo, there is no need for this. We only want to expose the base class (`Machine`) anyways. So all the other classes are OK to be hidden from SWIG (just need to port all python-undocumented examples as they of course will fail).